package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "leaflet-spin",
-  "version": "1.1.0",
```

Comment:
As we try to follow [semver](https://semver.org/) recommendation, the kind of change in the PR may be at least a *minor upgrade* (not just a *patch upgrade*) because it's not just a patch of bugfix.